UnityAds/src/main/java/com/mopub/mobileads/UnityInterstitial.java

```diff
@@ -132,9 +134,7 @@ protected void show() {
             metadata.setOrdinal(++impressionOrdinal);
             metadata.commit();
 
-            UnityAds.addListener(UnityInterstitial.this);
-
-            UnityAds.show((Activity) mContext, mPlacementId);
+            UnityAds.show((Activity) mContext, mPlacementId, mUnityShowListener);
```

Comment:
We need to be sure above this to remove `UnityAds.isReady` from above, and in all other locations